### DISCUSSION TITLE
How to disable HyperHDR turning on ambilight at boot

### DISCUSSION THREAD

**Discussion by Chao-chao00** [awawa-dev/HyperHDR#935] [Q&A] [answered]
Hi I've been using HyperHDR for quite some time and it works great.

I manually control HyperHDR​ status (turning on or off the ambilight​ LED​ strip)​ and don't use the auto/manual signal detection because from times to times it fails and need to redo it. 
So I connected HyperHDR to Home Assistant and use a Zigbee button to turn on/off the ambilight manually and it works great. 

However I have a slightly annoying problem that occurs when there is a power outage. When the power is back the raspberry pi zero 2 w​ that hosts HyperHDR boots, HyperHDR autostart and it turns on the ambilight by default.

Is it possible to change that behavior and have HyperHDR not turning on the ambilight when it autostart?

Thank you.

**Comment by awawa-dev** [comment, marked as answer]
Hi
Try this option:
![obraz](https://github.com/user-attachments/assets/baa8c66c-46a4-4284-8d83-bbe3fa8a3afc)

**Reply by Chao-chao00** [reply]
> Hi Try this option: ![obraz](https://private-user-images.githubusercontent.com/69086569/365476648-baa8c66c-46a4-4284-8d83-bbe3fa8a3afc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU4NTE0NDcsIm5iZiI6MTcyNTg1MTE0NywicGF0aCI6Ii82OTA4NjU2OS8zNjU0NzY2NDgtYmFhOGM2NmMtNDZhNC00Mjg0LThkODMtYmJlM2ZhOGEzYWZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA5VDAzMDU0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFkOTY2Y2MwYmM3ZjNiOTRlNDQ2NzY3ZjczNTdhNDZmYTMxNmYwMTY0NWI5Mzg3YzNjOGY3YzIyZjY5NzI3MzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.CXSU9ciY1NbqCF2MzfZY-fOQrOjdPXgdJtnGeXAhPts)

Hi thanks for answering, I didn't see this option. I am using the older version v19.0.0.0

I guess this is a new option available in v20.0.0.0

Do I have to reflash the sd card with v20.0.0.0 and reconfigure HyperHDR or is there an easier way to update?

Best regards.

**Reply by awawa-dev** [reply]
Before you start doing anything, make a backup of your settings (general tab). Then you can choose any installation method described in the wiki, even starting from scratch by writing latest HyperHDR SD image.

**Reply by Chao-chao00** [reply]
I updated to v20.0.0.0 by making a backup of my settings, flashing HyperHDR Rpi Zero 2 W image and uploading settings.
Thank you, it works.
